### PR TITLE
different lat/lon grid spacing

### PR DESCRIPTION
**Pull request by wpb-astro**
Hi Peter - skymapper is working beautifully for me! I have found it useful to set different grid spacings along the parallels / meridians - creating a pull request in case it could be useful. Thanks !

### CODE REVIEW

File: skymapper/map.py [hunk 1]
```diff
@@ -390,6 +392,8 @@ def grid(self, sep=30, parallel_fmt=None, meridian_fmt=None, lat_min=-90, lat_ma
             lon_max: maximum longitude for graticules
             **kwargs: styling of `matplotlib.lines.Line2D` for the graticules
         """
+        if sep_lat is None or sep_lon is None:
+            sep_lat = sep_lon = sep
```
Comment (pmelchior) [inline skymapper/map.py]: I like the functionality! But this implementation is problematic. If you set `set=30`, and `sep_lon=None`, then `sep_lat` will be overwritten to `sep` whether it was set or not.

I recommend doing this differently. Let's make `sep` either a number or a list of 2 numbers, which will the be unpacked into `sep_lat` and `sep_lon`.

Comment (wpb-astro) [reply skymapper/map.py]: Sure! I've implemented it that way instead - it should allow for any two-element list/tuple/array, etc.

Comment (pmelchior) [reply skymapper/map.py]: ok. it still doesn't remove the ambiguity between specifying `sep` *and* either `sep_lat` or `sep_lon`. I'll merge this now and clean it up on my end.

**Comment by pmelchior** [comment]
Ignore my last comment, I was looking at the wrong changes. This is exactly what I suggested. Thank you!

**Pull request merged**